site/content/3.11/develop/integrations/_index.md

```diff
@@ -13,12 +13,22 @@ low-level network communication.
 
 ## Spring Data
 
-The [**Spring Data integration**](spring-data-arangodb/_index.md) for ArangoDB lets you use
-ArangoDB as a database system in Spring-based Java applications.
+The [**Spring Data integration**](spring-data-arangodb/_index.md) for ArangoDB
+lets you use ArangoDB as a database system in Spring-based Java applications.
 
-- Online course: [Spring Data Tutorial](https://university.arangodb.com/courses/spring-data-tutorial)
+- [Tutorial](spring-data-arangodb/_index.md#get-started)
 - Repository: [github.com/arangodb/spring-data](https://github.com/arangodb/spring-data)
-- [Changelog](https://github.com/arangodb/spring-data/blob/master/ChangeLog.md#readme)
+- [Changelog](https://github.com/arangodb/spring-data/blob/main/ChangeLog.md)
+
+## Spring Boot Starter
+
+The [**Spring Boot Starter**](spring-boot-arangodb.md) for ArangoDB is a set of
+convenient dependency descriptors that you can include in your application based
+on the Spring framework.
```

Comment:
I'm not sure how helpful this description is for someone who hasn't used Spring

Reply:
Feel free to change it.